src/App.css

```diff
@@ -8,17 +28,33 @@
 }
 
 .App-header {
-  background-color: #222;
+  background-color: lime;
   height: 150px;
   padding: 20px;
-  color: white;
+  color: red;
 }
 
 .App-intro {
-  font-size: large;
+  font-size: 600%;
+  text-align: center;
+  background-color: lime;
+  color: red;
+
+}
+
+p {
+  font-size: 100%;
 }
 
-@keyframes App-logo-spin {
+button{
+  font-size:100%;
+  text-align: center;
+  background-color: lime;
+
+
+}
+
+/* @keyframes App-logo-spin {
```

Comment:
If you want to remove this, just delete it.  We have the older version in git and can always recover it there if we need to, so in general we shouldn't commit any commented-out code.